docs/README.md

```diff
@@ -1,30 +1,33 @@
 
-The vfxt.py script is a command-line tool for creating and managing Avere clusters in cloud-based virtual computing environments.
+The vfxt.py script is a command-line tool for creating and managing Avere clusters in cloud-based virtual computing environments. 
 
-This script has complete capabilities for creating and managing Avere vFXT clusters, including:
+This script has complete capabilities for creating and managing Avere vFXT clusters, including: 
 
 * Create a new Avere vFXT cluster - including creating the vFXT nodes that make up the cluster and configuring cloud storage as a backend core filer
-* Destroy existing clusters (including the vFXT nodes)
+* Destroy existing clusters (including the vFXT nodes) 
 * Create and add new nodes to a cluster
 * Basic cluster configuration tasks
 
-For ongoing cluster administration, use the Avere Control Panel. Read the Avere cluster [Configuration Guide](<https://azure.github.io/Avere/#operations>) for more details.
+For ongoing cluster administration, use the Avere Control Panel. Read the Avere cluster [Configuration Guide](<http://library.averesystems.com/#operations>) for more details. 
 
-**NOTE:**
-A previous version of this product supported AWS and GCE environments in addition to Microsoft Azure. These capabilities are deprecated. Some commands related to those capabilities still appear in this documentation. Make sure you are using the specific commands documented for Microsoft Azure.
+The vfxt.py script can be used with any of the cloud computing providers that Avere OS supports. Environment setup requirements are different for the different platforms, and the exact commands available vary by cloud computing provider.
 
-This document gives a basic overview of the vfxt.py script and its options. However, setting up a cloud project and configuring it to provide an Avere vFXT cluster includes many more steps than are documented here. Project creation, identity and access management, networking, quota and billing concerns, security, and many other topics are explained in detail in the [Avere vFXT Installation Guide](<https://aka.ms/averedocs>).
+This document gives a basic overview of the vfxt.py script and its options. It includes information about commands specific to Microsoft Azure, Amazon Web Services, and Google Cloud Platform/Google Compute Engine cloud services. However, setting up a cloud project and configuring it to provide an Avere vFXT cluster includes many more steps than are documented here. Project creation, identity and access management, networking, quota and billing concerns, security, and many other topics are explained in detail in the Avere vFXT Installation Guide customized for your cloud provider. Read the complete details here: 
 
-The command `vfxt.py --help` gives a full list of command options, including provider-specific functionality.
+* [vFXT Installation Guide for Amazon Web Services](<http://library.averesystems.com/#vfxt>) 
```

Comment:
lines 18-21 shouldn't be here and line 22 needs revision to avoid mentioning provider-specific functionality